coralproject/talk
# [CORL-1517] Use relay to clear Done answered comments

## What does this PR do?

Replaced the `onRemoveAnswered` callbacks in the unanswered questions tab that were reloading the whole stream to instead use a simple mutation that updates the relay cache.

## What changes to the GraphQL/Database Schema does this PR introduce?

None

## How do I test this PR?

- Switch the stream to QA mode
- Ask a question as a commenter
- Using the expert account, view the "Unanswered" tab
- Answer the new question
- See the "Done" button show up
- Click Done
- See that the comment disappears and the comment counts update correctly
